MetaCell/salk-interactive-atlas
# Feature/186

Closes #186 

Implemented solution: 
- Adapts csv reader to read cells from the newly generated individual population csv (cells_with_labels_<Population_Name>) instead of the previous common transformed_cells.
- Population name now comes from the input data
- Uses cordmap's get_populations_from_file to know how many populations to generate from the input file
- For each population we trigger the execute_generate_population_cells_workflow which when finished will trigger execute_generate_population_static_files_workflow
- Removes 'Add another set of files from the frontend'
- Updates frontend's Cell model

:lantern: Breaking-Change label :lantern::
Previous populations won't show cells due to the new format used. We can regenerate them with a migration bug given the context we are (no production servers yet) I did not include them in the PR

:lantern: Alert:Deployment label :lantern::
Locally I had to increase my minikube memory from 6GB to 12GB in order to support the new parallel multiple population creation


How to test this PR:

1. Create an experiment from a input file with multiple populations (f.e. V1_Sp8_Cord3):

https://user-images.githubusercontent.com/19196034/192758895-6cd1ba08-f8d9-4ce0-9adc-7cd199c17d4f.mp4

2. Confirm multiple populations were created:

https://user-images.githubusercontent.com/19196034/192758899-94e6af3d-3719-4843-a7f8-b2875782fe69.mp4


### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):
